components/feature/downloads/src/main/java/mozilla/components/feature/downloads/ext/DownloadState.kt

```diff
@@ -35,6 +35,11 @@ internal fun DownloadState.withResponse(headers: Headers, stream: InputStream?):
         contentType = headers[CONTENT_TYPE]
     }
 
+    var contentLength = this.contentLength
```

Comment:
This appears to duplicate line 50: `contentLength ?: headers[CONTENT_LENGTH]?.toLongOrNull()`

Reply:
Fixed :)